Readme.md

```diff
@@ -47,7 +47,10 @@ app.keys = ['some secret hurr'];
 
 var CONFIG = {
   key: 'koa:sess', /** (string) cookie key (default is koa:sess) */
-  maxAge: 86400000, /** (number) maxAge in ms (default is 1 days) */
+  /** (number) maxAge in ms (default is 1 days) */
+    maxAge: 86400000,
+  /** string: 'session' will result in a cookie that expires when session/browser is closed */
+    maxAge: 'session',
```

Comment:
must add a warning message here: if set maxAge to `session`, once the cookie was stolen, the cookie will never expired.